doc/pages/administrator/030-installation_kubernetes.md

```diff
@@ -86,17 +86,67 @@ kubectl delete deployments.app -n kube-system astarte-operator
 **DO NOT** delete Astarte's CRDs! This will lead to the deletion of the entire Astarte deployment
 with a consequent data loss.
 
-To restore the Operator's functionalities with v1.0, all you have to do is following the install
-instructions, i.e.:
+Please, before proceeding to the next steps make sure the requirements listed in the
+[Prerequisites](./020-prerequisites.html#Prerequisites) section are satisfied (e.g. `cert-manager`
+installation).
+
+To restore the Operator's functionalities with v1.0, the first step is adding the Astarte Operator's
+Helm chart repository:
 ```bash
 helm repo add astarte https://helm.astarte-platform.org
 helm repo update
-helm install astarte-operator astarte/astarte-operator --version 1.0.0-alpha.1
 ```
 
-Please note that version `1.0.0-alpha1` is a safe landing version to perform the upgrade. After
-the successful installation, please upgrade to a more recent version following the instructions
-outlined in the [previous section](#upgrading-the-operator-v1-0).
+Then, let's set some environment variables that will come handy for the upcoming migration
+procedure:
++ `DIR` is the target directory in which the chart templates will be generated,
++ `RELEASE_NAME` is the name of the Astarte Operator deployment,
++ `RELEASE_NAMESPACE` is the namespace in which the Astarte Operator will reside.
+
+You can change the following values at your preference. However, the following values should be ok
+for the large majority of use cases.
+
+Note that you are responsible for checking that `RELEASE_NAMESPACE` exists within the cluster.
+```bash
+export DIR=/tmp
```

Comment:
isn't `DIR` a little bit too generic? I see this disrupting a user's environment pretty easily, let's pick some less generic names (e.g.: prepend `ASTARTE_` to all env vars)